syntax="proto3";

enum ProtoErrorType {
	PROTO_ERROR_NONE								= 0;
	PROTO_ERROR_MSG_DATA_INVALIAD					= 1;
	PROTO_ERROR_LOGIN_ACCOUNT_OR_PASSWORD_INVALID	= 2;
	PROTO_ERROR_LOGIN_VERIFY_ACCOUNT_TIMEOUT		= 3;
	PROTO_ERROR_LOGIN_REPEATED						= 4;
	PROTO_ERROR_LOGIN_DATA_INVALID					= 5;
}
